renderer/configuration/robotouille_config.json

```diff
@@ -1,12 +1,12 @@
 {
-    "version": "1.0.0",
+    "version": "1.0.1",
 
     "player": {
         "robot": {
-            "front": "robot_front.png",
-            "back": "robot_back.png",
-            "left": "robot_left.png",
-            "right": "robot_right.png"
+            "front": ["robot_front_1.png", "robot_front_2.png", "robot_front_3.png", "robot_front_4.png"],
```

Comment:
This can be a temporary solution for now though `robot_front.png` and the other spritesheets are meant to be used directly by programmatically splitting it into rows and columns and choosing a grid cell based on the frame to display. This will require a new class and could be bundled in a separate PR for creating a dedicated animation class so let's make an issue for this.

Reply:
Issue made